x-pack/plugins/alerting/server/task_runner/execution_handler.ts

```diff
@@ -201,7 +201,12 @@ export class ExecutionHandler<
           if (isSummaryActionPerRuleRun(action) && !this.hasAlerts(alerts)) {
             continue;
           }
-          const summarizedAlerts = await this.getSummarizedAlerts({ action, spaceId, ruleId });
+          const summarizedAlerts = await this.getSummarizedAlerts({
+            action,
+            spaceId,
+            ruleId,
+            excludedAlertInstanceIds: this.rule.mutedInstanceIds,
```

Comment:
nit: `this.rule.mutedInstanceIds` could be used directly in the method

Reply:
That's true, I made the change in https://github.com/elastic/kibana/pull/147664/commits/232e81c03380fd323184518f9da2f2f8b6c6620a. Thanks!